src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/DataGuidanceFileServiceTests.cs

```diff
@@ -136,7 +175,10 @@ await using (var contentDbContext = InMemoryContentDbContext(contextId))
         [Fact]
         public async Task CreateDataGuidanceFile_UploadThrows()
         {
-            var release = new Release();
+            var release = new Release
+            {
+                MetaGuidance = "Test guidance"
```

Comment:
I can't see this text being checked for later on in the test.  Is it not important to test the actual output based on this text value, or are you happy that it's covered elsewhere?

Reply:
Good point, I originally had a check in `DataGuidanceFileWriter` to ensure there was guidance, but I removed it when I moved it to `Content.Services` so that it could work with older releases that may not have `MetaGuidance` defined.

I think my intention was to put the check in `DataGuidanceFileService` but I must have forgotten! Thanks!